torch_geometric/explain/config.py

```diff
@@ -75,6 +76,9 @@ class ExplainerConfig(CastMixin):
 
                 - :obj:`"both"`: Will mask both the node and its attributes..
 
```

Comment:
Is both outdated? Doesn't seem to be referenced anymore.

Reply:
I understand. Maybe change the order to be `object`, `attribute`, `common_attribute`, `both` in the Enum and ExplainerConfig docstring. Then describe both as enabling `object` and `attribute`. Is there a way to enable `object` and `common_attribute`?

Reply:
yes you're right, I'll update the docs, it can be make clearer

>  Is there a way to enable object and common_attribute?
Not yet, not sure how to do it properly. Another mask type ?